rosdep/python.yaml

```diff
@@ -8519,6 +8535,18 @@ python3-weasyprint-pip:
   ubuntu:
     pip:
       packages: [weasyprint]
+python3-webargs:
+  arch: [python-webargs]
+  debian:
+    pip:
```

Comment:
Please change this to explicitly target bullseye and buster.

Reply:
resolved in bf1169b2bc4b36c3c5fb44d670d279a9a62bee5d